isis/tests/FunctionalTestsLeisa2isis.cpp

```diff
@@ -63,18 +63,19 @@ TEST(leisa2isisTest, leisa2isisTestDefault) {
 
   // BandBin Group
   // Check size, first, 2 middle, and last values? Enough?
+  PvlGroup &bandbin = isisLabel->findGroup("BandBin", Pvl::Traverse);
   ASSERT_EQ(bandbin["Center"].size(), 256);
   ASSERT_EQ(bandbin["Width"].size(), 256);
   ASSERT_EQ(bandbin["OriginalBand"].size(), 256);
 
   ASSERT_DOUBLE_EQ(bandbin["Center"][0].toDouble(), 2.4892);
-  ASSERT_DOUBLE_EQ(bandbin["Center"][64].toDouble(), 0);
-  ASSERT_DOUBLE_EQ(bandbin["Center"][128].toDouble(), 0);
+  ASSERT_DOUBLE_EQ(bandbin["Center"][64].toDouble(), 1.9784);
+  ASSERT_DOUBLE_EQ(bandbin["Center"][128].toDouble(), 1.572);
   ASSERT_DOUBLE_EQ(bandbin["Center"][255].toDouble(), 2.0898);
 
   ASSERT_DOUBLE_EQ(bandbin["Width"][0].toDouble(), 0.011228);
-  ASSERT_DOUBLE_EQ(bandbin["Width"][64].toDouble(), 0);
-  ASSERT_DOUBLE_EQ(bandbin["Width"][128].toDouble(), 0);
+  ASSERT_DOUBLE_EQ(bandbin["Width"][64].toDouble(), 0.008924);
+  ASSERT_DOUBLE_EQ(bandbin["Width"][128].toDouble(), 0.007091);
```

Comment:
I'm assuming these values are correct as all the original leisa2isis app tests are now passing after the bug fix.

Reply:
I guess I didn't push my last commit. Very sorry about this and thank you for doing the cleanup.